recipe/migrations/cuda130.yaml

```diff
@@ -0,0 +1,64 @@
+migrator_ts: 1755016036
+__migrator:
+  use_local: true
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    0
+  paused: false
+  override_cbc_keys:
+    - cuda_compiler_stub
+  check_solvable: false
+  primary_key: cuda_compiler_version
+  ordering:
+    cuda_compiler_version:
+      - 12.4
+      - 12.6
+      - 12.8
+      - None
+      - 12.9
+      - 13.0
+      # to allow manual opt-in for CUDA 11.8, see
+      # https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/7472
+      # must be last due to how cuda_compiler ordering in that migrator works
+      - 11.8
+  wait_for_migrators:
+    - cuda129
```

Comment:
On second thought, this should very likely use `operation: key_add`. cuda129.ymal updates in place, but cuda13.yaml extends the zip.

Reply:
The documentation isn't great, but I though `key_add` was for adding new keys e.g. `c_compiler_version` not versions themselves e.g. `15`. Let me try again to get a separate migrator for CUDA 13.0. Otherwise, the current approach of this PR is to immediately close the CUDA 12.9 migrator and open this one which does both 12.9 and 13.0 simultaneously.

Reply:
`key_add` seems to be working how we want. You can apply the 130 migrator on top of the 129 migrator and it keeps both.

5b0c52a3370e75258a60028c95658475f57f4bcf